src/image_builder.py

```diff
@@ -139,6 +139,15 @@ def image_builder(buildspec):
         if "labels" in image_config:
             labels.update(image_config.get("labels"))
 
+        # Adding standard labels to all images
+        labels["com.amazonaws.sagemaker.dlc.framework"] = str(BUILDSPEC["framework"])
+        labels["com.amazonaws.sagemaker.dlc.framework_version"] = str(BUILDSPEC["version"])
+        labels["com.amazonaws.sagemaker.dlc.container_type"] = str(image_config["device_type"])
+        # tag_python_version in the buildspec looks like pyXY. We need to convert it to X.Y
+        labels["com.amazonaws.sagemaker.dlc.py_version"] = \
```

Comment:
can we add a separate label to identify a special flavor, like "huggingface" or "habana"?

Also, do we log "job_type" (training/inference)?